nc_controllers/app.controller.js

```diff
@@ -15,9 +15,16 @@ exports.getTopics = (req, res, next) => {
 
 exports.getArticles = (req, res, next) => {
   const { article_id } = req.params;
-  allArticles(article_id)
+  singleArticle(article_id)
     .then((article) => {
       res.status(200).send({ article });
     })
     .catch(next);
 };
+
+exports.getAllArticles = (req, res, next) => {
+  allArticles().then((articles) => {
```

Comment:
it's good convention to name functions as verbs to show what they do - e.g. here we could call this readArticles or fetchArticles